apps/src/templates/CensusForm.jsx

```diff
@@ -1,9 +1,13 @@
 import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
 import Checkbox from './forms/Checkbox';
 import {UnconnectedCensusFollowUp as CensusFollowUp} from './CensusFollowUp';
 import Button from './Button';
 import color from "../util/color";
 import i18n from "@cdo/locale";
+import '../sites/studio/pages/schoolInfo';
+import ProtectedStatefulDiv from '../templates/ProtectedStatefulDiv';
+require('selectize');
```

Comment:
Does requiring 'selectize' apply changes globally? (I suspect yes). If so, we should probably do this at the top level entry point similar to how we have it here https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/sites/studio/pages/code-studio.js#L36